torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -1486,7 +1486,8 @@ at::Tensor XLANativeFunctions::frac(const at::Tensor& self) {
 
 at::Tensor XLANativeFunctions::gather(const at::Tensor& self, int64_t dim,
                                       const at::Tensor& index,
-                                      bool /* sparse_grad */) {
+                                      bool /* sparse_grad */,
+                                      bool /* unique_indices */) {
```

Comment:
PyTorch added this parameter to `gather` since their implementation of `gather` calls their `scatter`. However, PT/XLA does not. PT/XLA directly calls `xla::Gather`.

Reply:
hmm, does this means pt/xla gather can handle `non-unique-index`?

Reply:
Good point, I was looking into this before a bit. According to XLA's documentation, there is no mention of non-deterministic behavior (https://www.tensorflow.org/xla/operation_semantics#gather). However, to be safe, I've also added the `XLA` check to `gather` in the new commit.